dockerhq.sh

```diff
@@ -80,7 +80,8 @@ case $key in
         web_runner run --rm web python manage.py migrate $@
         ;;
     runserver)
-        web_runner run --service-ports web $@
+        web_runner run --service-ports -e CUSTOMSETTINGS="docker.localsettings_docker" \
```

Comment:
Specifying these localsettings means that if you want to change a setting you have to edit that localsettings file and not your normal `localsettings.py` which isn't great since `docker.localsettings_docker` is tracked by git.